hungarian/README.md

```diff
@@ -0,0 +1,139 @@
+
+* `tables/hu-hu-g2.ctb`: this table contains the Hungarian grade 2
+  braille rules, which is used in literary books, braille news papers,
+  etc. Hungarian grade 2 braille has two variations, small, fever
+  abbreviations containing rule set (this variation containing only 45
+  abbreviation rules), and a larger variant, which is used for
+  literary books and braille printing documents.
+
+In 2017, the larger variant was implemented.
+
+Official table maintainer from Liblouis community in Hungary: Attila
+Hammer from [IT Foundation for the Visually Impaired -
+Hungary](http://www.infoalap.hu). Attila is subscribed in Liblouis
+list. If anybody have question the Hungarian braille tables related,
+feel free ask a question in developer community.
+
+Before modifying the tables please ask the table maintainer. If you
+would like to do bigger changes in the Hungarian braille tables,
+please do not commit the changes to the [Liblouis
+repository](https://github.com/liblouis/liblouis) until all test cases
+are passing, because Hungarian language is a very complex language.
+
+If a maintenance change related have a question, please write e-mail
+to the table maintainer with following addresses:
+
+* Full official work time used e-mail address, with available the
+  Hungarian braille tables: hammer.attila at infoalap dot hu
+* Private e-mail address, with Attila subscribed into Liblouis mailing
+  list: hammera at pickup dot hu
+
+## Hungarian language awailable downloadable documents
+
+The Hungarian Braille Committee official sub page is following:
+
+[The Hungarian Braille Committee official sub page](https://www.mvgyosz.hu/tudjon-meg-tobbet-hu/braille-iras/)
+
+Following two documents are standard braille embossers USA braille
+character set compatible, windows CRLF line terminator wrapped
+documents, with containing the Hungarian braille alphabet. Without
+changes, this documents are free usable:
+
+* [Braille Alphabet Book (first part, USA BRF document, formatted with Windows line wrap)](https://www.mvgyosz.hu/wp-content/uploads/2020/04/braille-%C3%A1b%C3%A9c%C3%A9sk%C3%B6nyv01.brf)
+* [Braille Alphabet Book (second part, USA BRF document, formatted with Windows line wrap)](https://www.mvgyosz.hu/wp-content/uploads/2020/04/braille-%C3%A1b%C3%A9c%C3%A9sk%C3%B6nyv02.brf)
+* [Hungarian language used braille dots (an EXCEL table, with containing in 12 worksheet the various braille rules)](https://www.mvgyosz.hu/wp-content/uploads/2020/04/magyar-braille-%C3%ADr%C3%A1s-jelei.xls)
+* [Summary collection of decisions concerning Braille adopted by the presidency of the Hungarian National Association of the Blind and Visually Impaired](https://www.mvgyosz.hu/wp-content/uploads/2020/04/braille-%C3%ADr%C3%A1s-eln%C3%B6ks%C3%A9gi-hat%C3%A1rozatok.docx)
+* [The medicine boxes braille standards collection documents (dot sizes, line length, etc.)](https://www.mvgyosz.hu/wp-content/uploads/2020/04/braille-gy%C3%B3gyszercsomagol%C3%A1sokon.doc)
```

Comment:
When available, could you please add some more info (author, title, year, publisher, ISBN, etc.) about these documents?

Also, I think it might be useful to (if allowed) put copies of the documents on our website instead of linking to them. The reason is because there is always a risk that links become dead after some time.